marble-table/src/test/java/org/apache/calcite/table/TypeConvertUtilTest.java

```diff
@@ -0,0 +1,158 @@
+package org.apache.calcite.table;
+
+import org.apache.calcite.util.NlsString;
+import org.junit.Assert;
+import org.junit.Test;
+
```

Comment:
remove blank line and put imports in alphabetical order

Reply:
This is the IntelliJ formatting preference. Switching to the commonly used Eclipse format checker is effectively the same except that the Java classes come first. Both leave a blank line between groups of imports, and this also matches other test files in the project. Within these groups, the imports are in alphabetical order.